scalability_and_performance/telco_core_ref_design_specs/telco-core-rds.adoc

```diff
@@ -0,0 +1,228 @@
+:_mod-docs-content-type: ASSEMBLY
+:telco-core:
+[id="telco-core-ref-design-specs"]
+= Telco core reference design specifications
+include::_attributes/common-attributes.adoc[]
+:context: telco-core
+
+toc::[]
+
+The telco core reference design specification (RDS) configures an {product-title} cluster running on commodity hardware to host telco core workloads.
+
+include::modules/telco-core-rds-product-version-use-model-overview.adoc[leveloffset=+1]
+
+include::modules/telco-core-about-the-telco-core-cluster-use-model.adoc[leveloffset=+1]
+
+include::modules/telco-ran-core-ref-design-spec.adoc[leveloffset=+2]
+
+include::modules/telco-deviations-from-the-ref-design.adoc[leveloffset=+2]
+
+include::modules/telco-core-common-baseline-model.adoc[leveloffset=+1]
+
+include::modules/telco-core-cluster-common-use-model-engineering-considerations.adoc[leveloffset=+1]
+
+include::modules/telco-core-application-workloads.adoc[leveloffset=+2]
+
+include::modules/telco-core-signaling-workloads.adoc[leveloffset=+2]
+
+[id="telco-core-rds-components"]
```

Comment:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings
```suggestion
[id="telco-core-rds-components_{context}"]
```